src/libcore/iter/range.rs

```diff
@@ -367,6 +374,21 @@ impl<A: Step> Iterator for ops::RangeInclusive<A> {
         self.end.replace_zero();
         None
     }
+
+    #[inline]
+    fn last(self) -> Option<A> {
+        Some(self.end)
+    }
+
+    #[inline]
+    fn min(self) -> Option<A> {
+        Some(self.start)
+    }
+
+    #[inline]
+    fn max(self) -> Option<A> {
+        Some(self.end)
```

Comment:
Shouldn't this and `last` return `None` once the iterator is exhausted?

Reply:
Yep, it definitely should.